app/services/symphony_client.rb

```diff
@@ -172,6 +194,29 @@ def get_all_locations
 
   private
 
+    def patron_address(params)
+      params.permit(SAFE_PATRON_ADDRESS_FIELDS)
+        .to_h
+        .transform_keys(&:camelize)
+        .transform_keys(&:upcase)
```

Comment:
```suggestion
        .transform_keys(&:camelize)
        .transform_keys(&:upcase)
```
do we need the `camelize`?

Reply:
yeah. 

`blah_blah` becomes `blahBlah` becomes `BLAHBLAH`

Reply:
maybe there's a more clever way to do this 🤔 I didn't spend a lot of time on it

Reply:
ok makes sense, I did not think about underscore 👍